Makefile

```diff
@@ -149,55 +149,42 @@ e2e-dependencies:
 KIND_NODE_VERSION := kindest/node:v$(KUBERNETES_VERSION)
 
 e2e-create-local-registry:
-	if [ "$$(docker inspect -f '{{.State.Running}}' "registry" 2>/dev/null || true)" ]; then docker stop registry && docker rm registry; fi
-	docker pull ${LOCAL_REGISTRY_IMAGE}
-	docker run -d \
-		-p 5000:5000 \
-		--restart=always \
-		--name registry \
-		${LOCAL_REGISTRY_IMAGE}
-
-	rm -rf .staging
-	mkdir .staging
-	echo 'FROM alpine\nCMD ["echo", "all-in-one image"]' > .staging/Dockerfile
-	docker build -t ${LOCAL_TEST_REGISTRY}/all:v0 .staging
-	docker push ${LOCAL_TEST_REGISTRY}/all:v0
-
-e2e-create-local-registry-auth:
 	rm -rf ~/auth
 	mkdir ~/auth
 	docker run --entrypoint htpasswd httpd@sha256:dd993a2108430ec8fdc4942f791ccf9b0c7a6df196907a80e7e8a5f8f1bbf678 -Bbn ${LOCAL_TEST_REGISTRY_USERNAME} ${LOCAL_TEST_REGISTRY_PASSWORD}> ~/auth/htpasswd
 
-	if [ "$$(docker inspect -f '{{.State.Running}}' "registry-auth" 2>/dev/null || true)" ]; then docker stop registry-auth && docker rm registry-auth; fi
+	if [ "$$(docker inspect -f '{{.State.Running}}' "registry" 2>/dev/null || true)" ]; then docker stop registry && docker rm registry; fi
 	docker pull ${LOCAL_REGISTRY_IMAGE}
 	docker run -d \
-		-p 5001:5000 \
+		-p 5000:5000 \
 		--restart=always \
-		--name registry-auth \
+		--name registry \
 		-v ${HOME}/auth:/auth \
 		-e "REGISTRY_AUTH=htpasswd" \
 		-e "REGISTRY_AUTH_HTPASSWD_REALM=Registry Realm" \
 		-e REGISTRY_AUTH_HTPASSWD_PATH=/auth/htpasswd \
 		${LOCAL_REGISTRY_IMAGE}
 
-	docker login -u ${LOCAL_TEST_REGISTRY_USERNAME} -p ${LOCAL_TEST_REGISTRY_PASSWORD} ${LOCAL_TEST_REGISTRY_AUTH}
+	docker login -u ${LOCAL_TEST_REGISTRY_USERNAME} -p ${LOCAL_TEST_REGISTRY_PASSWORD} ${LOCAL_TEST_REGISTRY}
 	oras login \
 		-u ${LOCAL_TEST_REGISTRY_USERNAME} \
 		-p ${LOCAL_TEST_REGISTRY_PASSWORD} \
-		${LOCAL_TEST_REGISTRY_AUTH}
+		${LOCAL_TEST_REGISTRY}
 
 	rm -rf .staging
 	mkdir .staging
+	echo 'FROM alpine\nCMD ["echo", "all-in-one image"]' > .staging/Dockerfile
+	docker build -t ${LOCAL_TEST_REGISTRY}/all:v0 .staging
+	docker push ${LOCAL_TEST_REGISTRY}/all:v0
```

Comment:
nit: I feel we can move it out to a separate target as it does not create a local registry.

Reply:
Yes this makes sense. I see that your azure e2e test PR does this? Should I just leave it as is for now?